tests/YotiClientTest.php

```diff
@@ -1,11 +1,15 @@
 <?php
-use Yoti\YotiClient;
 
+use Yoti\YotiClient;
+use Yoti\Entity\Country;
+use Yoti\Entity\AmlAddress;
+use Yoti\Entity\AmlProfile;
+use Yoti\Http\AmlResult;
 
-define('PEM_FILE', __DIR__ . '/../src/sample-data/yw-access-security.pem');
-define('SDK_ID', '990a3996-5762-4e8a-aa64-cb406fdb0e68');
-define('YOTI_CONNECT_TOKEN', file_get_contents(__DIR__ . '/../src/sample-data/connect-token.txt'));
-define('INVALID_YOTI_CONNECT_TOKEN', 'sdfsdfsdasdajsopifajsd=');
+defined('PEM_FILE') || define('PEM_FILE', __DIR__ . '/../src/sample-data/yw-access-security.pem');
```

Comment:
How can this be defined already? And if it is, shouldn't we re-define it before running the following tests?

Reply:
The same constant is defined in `RestRequestTest` with the same value - because UNIT TEST is loading all the files in no particular order when you run the test - then you will end up with an errors if you don't add that condition.